### PR TITLE
Homework 4

### PR DESCRIPTION
**Pull request by VadymDemchenko**
Homework_4 done

### CODE REVIEW

File: HW03/homework_3.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+print("I love Python" * 42) #1
```
Comment (klymenok) [inline HW03/homework_3.py]: This file shouldn't be here. Remove it, please

**Review by klymenok — CHANGES_REQUESTED** [review]
Good job, but you should remove file with previous HW

File: HW03/homework_3.py (hunk 1 above)
Comment (VadymDemchenko) [reply HW03/homework_3.py]: directory and file are removed

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**